test/script/tskdbdiff.py

```diff
@@ -354,52 +406,55 @@ def normalize_db_entry(line, table, vs_parts_table, vs_info_table, fs_info_table
         table: a map from object ids to file paths.
     """
 
-    files_index = line.find('INSERT INTO "tsk_files"')
-    path_index = line.find('INSERT INTO "tsk_files_path"')
-    object_index = line.find('INSERT INTO "tsk_objects"')
-    report_index = line.find('INSERT INTO "reports"')
-    layout_index = line.find('INSERT INTO "tsk_file_layout"')
-    data_source_info_index = line.find('INSERT INTO "data_source_info"')
-    ingest_job_index = line.find('INSERT INTO "ingest_jobs"')
+    files_index = line.find('INSERT INTO "tsk_files"') > -1 or line.find('INSERT INTO tsk_files ') > -1
+    path_index = line.find('INSERT INTO "tsk_files_path"') > -1 or line.find('INSERT INTO tsk_files_path ') > -1
+    object_index = line.find('INSERT INTO "tsk_objects"') > -1 or line.find('INSERT INTO tsk_objects ') > -1
+    report_index = line.find('INSERT INTO "reports"') > -1 or line.find('INSERT INTO reports ') > -1
+    layout_index = line.find('INSERT INTO "tsk_file_layout"') > -1 or line.find('INSERT INTO tsk_file_layout ') > -1
+    data_source_info_index = line.find('INSERT INTO "data_source_info"') > -1 or line.find('INSERT INTO data_source_info ') > -1
+    ingest_job_index = line.find('INSERT INTO "ingest_jobs"') > -1 or line.find('INSERT INTO ingest_jobs ') > -1
```

Comment:
Please add a comment explaining that this code takes into account the differnece between the PostgreSQL and SQLite syntax.